data/filters/templates/youtube-shorts.yaml

```diff
@@ -17,18 +17,11 @@ template: |
   www.youtube.com##ytd-browse[page-subtype="home"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-rich-item-renderer)
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-grid-video-renderer,ytd-rich-item-renderer)
   www.youtube.com##ytd-search .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
-  {{! Suggestions, old layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-compact-video-renderer,ytd-shelf-renderer)
-  {{! Suggestions, new layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer ytd-reel-shelf-renderer
   {{! Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections }}
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] ytd-video-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-item-section-renderer)
-  {{! Channels homepage }}
-  www.youtube.com##ytd-browse[page-subtype="channels"] #contents.ytd-reel-shelf-renderer:upward(ytd-item-section-renderer)
-  {{! Trending section }}
-  www.youtube.com##ytd-browse[page-subtype="trending"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
```

Comment:
Was outdated, it's a `ytd-reel-shelf-renderer` now

Reply:
The first rule (channel homepage) still works, but is already covered by `ytd-reel-shelf-renderer`
See: 
![afbeelding](https://github.com/letsblockit/letsblockit/assets/81161435/18b5f2cc-d5d0-473f-861c-b7b14efbbedd)

Only the rule for the trending section is indeed outdated

Thus both can be removed, although the first rule still works, but is not needed anymore.